common/makefile.mk

```diff
@@ -63,7 +63,7 @@ overlay: src
 # Apply quilt patches to the upstream source tree
 .PHONY: patch
 patch: src
-	quilt --quiltrc=/dev/null applied > /dev/null 2>&1 || quilt --quiltrc=/dev/null push -a
+	! quilt --quiltrc=/dev/null unapplied > /dev/null 2>&1 || quilt --quiltrc=/dev/null push -a
```

Comment:
Is this a previous typo you're fixing here or something new?  I'm guessing the former, in which case YIKES!